dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -86,11 +87,20 @@ const realTimeConfig = (
 		'gdpr_consent=CONSENT_STRING',
 	].join('&');
 
+	const urls = [
+		usePrebid ? prebidURL : '',
+		usePermutive ? permutiveURL : '',
+	].filter((url) => url);
+
+	const vendors = useAmazon
+		? {
+				aps: { PUB_ID: '3722', PARAMS: { amp: '1' } },
```

Comment:
Minor non-blocking point -  but it looks like most of the consts in `realTimeConfig` are static or mostly static so they could be moved out of the function entirely or moved out and then used as the basis for new objects (by spreading) when needed.

Reply:
Good point. There's probably more that can be done here but I've moved a few definitions outside the method